libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Preds/PParams.hs

```diff
@@ -106,9 +120,16 @@ pParamsPreds p =
             [ extract (maxTxExUnits p) (pparams p)
             , extract (collateralPercentage p) (pparams p)
             ]
-          PParamsBabbageToConway ->
+          PParamsBabbageToBabbage ->
+            [ extract (maxTxExUnits p) (pparams p)
+            , extract (collateralPercentage p) (pparams p)
+            ]
+          PParamsConwayToConway ->
             [ extract (maxTxExUnits p) (pparams p)
             , extract (collateralPercentage p) (pparams p)
+            , extract (drepDeposit p) (pparams p)
+            , extract (drepDeposit p) (pparams p)
```

Comment:
It doesn't look like it was intended to be listed twice, no?
```suggestion
```

Reply:
done